package/MDAnalysis/visualization/streamlines.py

```diff
@@ -152,8 +154,10 @@ def split_grid(grid, num_cores):
     return [list_square_vertex_arrays_per_core, list_parent_index_values, current_row, current_column]
```

Comment:
This is where you would be returning a tuple instead of a list.
```suggestion
    return (list_square_vertex_arrays_per_core, list_parent_index_values, current_row, current_column)
```

Assuming it passes the test, it means you should also update the docstring and ass a `.. version-changed::` note.